lib/sord/rbi_generator.rb

```diff
@@ -31,7 +31,7 @@ def object_count
     # @option options [Integer] break_params
     # @option options [Boolean] replace_errors_with_untyped
     # @option options [Boolean] comments
-    # @return [RbiGenerator]
+    # @return [void]
```

Comment:
I use void here because the FAQ recommends it: https://sorbet.org/docs/faq#how-do-i-write-a-signature-for-initialize

I wonder if we should always generate void for `initialize` methods 🤔